pkg/cmd/root/help_topic.go

```diff
@@ -6,6 +6,25 @@ import (
 )
 
 var HelpTopics = map[string]map[string]string{
+	"mintty": {
+		"short": "Information about using gh with MinTTY",
+		"long": heredoc.Doc(`
+			MinTTY is the terminal emulator that comes by default with Git
+			for Windows.  It has known issues with gh's ability to prompt a
+			user for input.
+
+			There are a few workarounds to make gh work with MinTTY:
+
+			- Reinstall Git for Windows. 
+			  Be sure to check the box "Enable experimental support for pseudo consoles".
```

Comment:
Nit: this should be in the same sentence, not two separate sentences, since it's paramount that people check the box when reinstalling.